bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -184,7 +185,7 @@ private func dummy() {
     _ = NSLocalizedString("Make your content stand out by adding images, gifs, videos, and embedded media to your pages.", comment: "")
     _ = NSLocalizedString("Media options", comment: "translators: %s: block title e.g: \"Paragraph\".")
     _ = NSLocalizedString("Mention", comment: "")
-    _ = NSLocalizedString("Minimum height", comment: "")
+    _ = NSLocalizedString("More support options", comment: "")
```

Comment:
- Addition
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/editor-help/index.native.js#L176:L176)

Looks like we missed this string, most likely this was caused due to the parsing issues when extracting the strings.